### PR TITLE
Fix/ios loading accessibility issue

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Currently, the loading screen does not get announced by VoiceOver on iOS.

# Solution
Use `role='progressbar'` + better focus management. 
Tested on:

- Safari on iPhone 7
- Safari on iPad
- Safari on Mac
- Chrome for Android
- Chrome on Mac.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Spinner/index.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+import { h, Component } from 'preact'
 import { compose } from '~utils/func'
 import { localised } from '../../locales'
 import style from './style.scss'
 
-const Spinner = ({ translate }) => {
-  return (
+class Spinner extends Component {
+  componentDidUpdate() {
+    this.container && this.container.focus()
```
Comment (ethan605) [inline src/components/Spinner/index.js:8]: IMHO it's worth adding an explanation here
```suggestion
    /**
     * Until iOS 14, the loading screen does not get announced by VoiceOver on iOS
     * This manual focus() trigger replaces `autoFocus` prop on the main `div`
     * to make sure it works properly on every platform.
     */
    this.container && this.container.focus()
```

**Review by ethan605 — CHANGES_REQUESTED** [review]

File: src/components/Spinner/index.js (hunk 1 above)
Comment (stefaniacardenas) [reply src/components/Spinner/index.js:8]: It's broken on iOS14 too. I will add comments as inline documentation

Comment (stefaniacardenas) [reply src/components/Spinner/index.js:9]: I've added comments, let me know your thoughts

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — APPROVED** [review]
LGTM

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1212-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1212-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**